Ports/schismtracker/package.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash ../.port_include.sh
+port=schismtracker
+version=20240529
+#workdir=SchismTracker-$version
+useconfigure=true
+files=(
+    "https://github.com/schismtracker/schismtracker/archive/$version.tar.gz#73c475b3344c460cbc543878f8c728af3c62fba11211604834d880c0a41a506e"
```

Comment:
Please use the official source code tarball instead of the GitHub-generated one. That allows us to not depend on `autotools` on the build host.